bigframes/dataframe.py

```diff
@@ -2666,9 +2666,10 @@ def to_gbq(
                 )
             if_exists = "replace"
 
-            temp_table_ref = bigframes.session._io.bigquery.random_table(
-                self._session._anonymous_dataset
-            )
+            temp_table_ref = self._session._random_table(
+                skip_cleanup=True
+            )  # the client code owns
+            # this table reference now, so skip_cleanup=True to not clean it up when we close the session
```

Comment:
I think the formatting got a little funky maybe because of `black`

```suggestion
            temp_table_ref = self._session._random_table(
                # The client code owns this table reference now, so skip_cleanup=True
                #  to not clean it up when we close the session.
                skip_cleanup=True,
            )
```